src/ORM/DataObject.php

```diff
@@ -2725,6 +2725,13 @@ public function dbObject($fieldName)
             ? $this->record[$fieldName]
             : null;
 
+        if (!$value && isset($this->record[$fieldName . '_Lazy'])) {
+            $tableClass = $this->record[$fieldName . '_Lazy'];
+            $this->loadLazyFields($tableClass);
+
+            $value = isset($this->record[$fieldName]) ? $this->record[$fieldName] : null;
```

Comment:
I think we do need to investigate how a `$field . '_Lazy'` can still be unset after calling `loadLazyFields`